### PR TITLE
#312 - adding sphinx build action

### PR DESCRIPTION
**Pull request by lyashevska**
Creating pull request to see whether this event triggers sphinx build.

### CODE REVIEW

**Comment by lyashevska** [comment]
@cwmeijer sphinx build now works as expected, ready for a review.

**Comment by maltelueken** [comment]
Great that you fixed the docs build! I have one suggestion: Instead of having a separate `requirements.txt` for the docs action, we could install the package with the dev requirements (this is what the user would do to build the docs) and add the docs requirements there.

File: .github/workflows/sphinx_build.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+        # this action updates pip, setuptools, and wheel before install
+      - name: Install Python dependencies
+        uses: py-actions/py-dependency-install@v4
+        with:
+           path: "docs/requirements.txt"
```
Comment (maltelueken) [inline .github/workflows/sphinx_build.yml:28]:
```suggestion
        run: pip install .[dev]
```

Comment (maltelueken) [reply .github/workflows/sphinx_build.yml:28]: In additiona we would need to add:
```
sphinx
sphinx-rtd-theme
```
to setup.py under `extras_require={'dev':}`

**Pull request merged**